ops-bedrock/docker-compose.yml

```diff
@@ -23,6 +26,28 @@ services:
       timeout: 5s
       retries: 5
       start_period: 10s
+# If you would like to use a light node for `celestia-node`, you can
+# uncomment the following lines. You should also comment out the `depends_on:`
+# section on lines 165-167. In order to use a light node, you will need to
+# sync the node first and fund it with TIA. This is so that the light node
+# can post data to Celestia. This `docker-compose.yml` file is set up to
+# look in your `$HOME` for the `.celestia-light-blockspacerace-0` directory.
```

Comment:
Docker throws a pretty loud warning on sharing home directory with the container.

It would be better to share just the light node directory, if it avoids this.

![2023-05-08_12-29](https://user-images.githubusercontent.com/132667/236915769-798d3f0e-7871-4025-baad-f452ab653200.png)

Reply:
sharing a light node directory makes little to no sense, as we would have to upload the `.celestia-light-blockspacerace-0/data` directory into the repo, which is about 12GB on BSR now. this is why I am providing guidance to the user to sync and fund their own light node.

in the documentation, i specify that the default is to run from your `$HOME` directory, which is also the default for `celestia-node`. so, if the user wants to change this, they can do so. however, to make this work out of the box easier for a non-advanced user, I think this is the best way to do this for now